docs/tutorials/setup-opentelemetry.md

```diff
@@ -46,23 +48,56 @@ const sdk = new NodeSDK({
 sdk.start();
 ```
 
-In the `index.ts`, import this file **at the beginning**:
+Your probably won't need all the instrumentations inside `getNodeAutoInstrumentations()` so make sure to
+check the [documentation](https://www.npmjs.com/package/@opentelemetry/auto-instrumentations-node) and tweak it properly.
 
-```typescript
-import './instrumentation'; // Setup the OpenTelemetry instrumentation
+It's important to setup the NodeSDK and the automatic instrumentation **before** importing any library.
 
-// other imports and backend init...
-```
+This is why we will use the nodejs [`--require`](https://nodejs.org/api/cli.html#-r---require-module)
+flag when we start up the application.
 
-It's important to setup the NodeSDK and the automatic instrumentation **before** importing any library.
+In your `Dockerfile` add the `--require` flag which points to the `instrumentation.ts` file
+
+```Dockerfile
+FROM node:18-bookworm-slim
+...
```

Comment:
Would there be other content here? I find this can confuse people so maybe we just leave a comment to that fact